### PR TITLE
Make printing human readable

### PR DESCRIPTION
**Pull request by Viterkim**
[sc-92152]

### CODE REVIEW

**Comment by Viterkim** [comment]
Looks like:
```
Using cache folder: /var/folders/3r/g17qgwr96r95f05ghj7z58_40000gn/T/rootfs-diff
Image size 68.10 MB (71409664 B) -> 71.90 MB (75390976 B) (size-diff: 3.80 MB (3981312 B), bsdiff: 0 B, minibsdiff: 0 B, minibsdiff-zstd: 0 B, vcdiff: 0 B, vcdiff-zstd: 0 B, zstd-diff: 0 B)
Grouped new files:
Single new files:
  usr/bin/disk-cleanup: 1.29 MB (1357852 B)
  usr/bin/ecu-sim: 1.14 MB (1190272 B)
  usr/bin/iface: 1.08 MB (1128504 B)
  usr/bin/drb-decoder: 837.94 KB (858052 B)
  etc/fstab.iwg26: 758 B
  etc/fstab.stim: 755 B
  lib/systemd/network/80-container-ve.network: 718 B
  lib/systemd/network/80-container-vz.network: 704 B
  lib/systemd/network/80-vm-vt.network: 672 B
  lib/systemd/network/80-container-host0.network: 645 B
  lib/systemd/network/99-default.link: 491 B
  lib/systemd/system/ip-can0-up.service: 259 B
  lib/systemd/system/ip-can1-up.service: 259 B
  etc/profile.d/ccupd.sh: 151 B
  lib/systemd/network/80-wired.network: 146 B
  etc/fw_env.iwg26: 110 B
  lib/systemd/network/80-wifi-ap.network.example: 101 B
  etc/fw_env.stim: 82 B
  lib/systemd/network/80-wifi-adhoc.network: 78 B
  lib/systemd/network/80-wifi-station.network.example: 64 B
  lib/systemd/network/90-wireless.network: 40 B
  lib/systemd/network/80-usb-test.network: 39 B
  lib/systemd/network/90-ethernet.network: 39 B
  lib/systemd/system-preset/98-ecu-sim.preset: 23 B
Grouped removed files:
Single removed files:
  usr/bin/sftp: 101.63 KB (104072 B)
  usr/libexec/sftp-server: 73.55 KB (75320 B)
  usr/bin/update-mfg.sh: 1.29 KB (1322 B)
  etc/fstab.imx6ul-emmc: 874 B
  etc/fstab.imx6ul-nand: 866 B
  etc/disk-cleanup.yaml: 567 B
  lib/systemd/system/update-mfg.service: 322 B
  lib/systemd/system/etc-environment.mount: 275 B
  lib/systemd/system/cc-readonly-fix.service: 213 B
  lib/systemd/system/reboot-data-reformat.service: 209 B
  lib/systemd/system/cc-hal-disk-cleanup.service: 194 B
  lib/systemd/system/cc-readonly-fix.timer: 146 B
  etc/fw_env.nand: 110 B
  etc/environment.iwg26-v1: 92 B
  etc/environment.iwg26-v2: 92 B
  etc/environment.iwg26-v2-vag: 92 B
  etc/environment.iwg26-v3: 92 B
  etc/environment.iwg26-v3-ag: 92 B
  etc/environment.iwg26-v3-ag-vag: 92 B
  etc/environment.stim-v1: 90 B
  etc/fw_env.emmc: 82 B
  lib/systemd/network/80-usb-client.network: 77 B
  lib/systemd/network/70-ethernet.network: 71 B
  lib/systemd/network/90-can.network: 71 B
  etc/env/iwg26-v1/cc-hal-disk-cleanup.conf: 70 B
  lib/systemd/system-preset/98-cc-environment.preset: 29 B
  lib/systemd/system-preset/98-update-mfg.preset: 26 B
  etc/environment: 0 B
Updated files:
  etc/build: 550 B -> 630 B, diff: 80 B (size-diff:80)
  etc/fstab.base-files: 650 B -> 758 B, diff: 108 B (size-diff:108)
  etc/fstab.qemu: 849 B -> 414 B, diff: -435 B (size-diff:-435)
  etc/issue: 33 B -> 63 B, diff: 30 B (size-diff:30)
  etc/issue.net: 30 B -> 60 B, diff: 30 B (size-diff:30)
  etc/nftables.conf: 914 B -> 863 B, diff: -51 B (size-diff:-51)
  etc/shadow: 690 B -> 690 B, diff: 0 B (size-diff:0)
  etc/ssh/authorized_keys: 942 B -> 924 B, diff: -18 B (size-diff:-18)
  etc/ssh/sshd_config: 3.05 KB (3119 B) -> 3.12 KB (3192 B), diff: 73 B (size-diff:73)
  usr/bin/cc-hal: 6.68 MB (7003464 B) -> 5.50 MB (5765996 B), diff: -1.18 MB (-1237468 B) (size-diff:-1237468)
  usr/bin/cc-info: 201.78 KB (206624 B) -> 1.01 MB (1058728 B), diff: 832.13 KB (852104 B) (size-diff:852104)
  usr/bin/ccupd: 196.32 KB (201032 B) -> 212.88 KB (217992 B), diff: 16.56 KB (16960 B) (size-diff:16960)
  usr/bin/data-reformat-ext4.sh: 42 B -> 42 B, diff: 0 B (size-diff:0)
  usr/bin/usb-ethernet-gadget.sh: 7.23 KB (7405 B) -> 6.46 KB (6620 B), diff: -785 B (size-diff:-785)
  usr/lib/os-release: 106 B -> 201 B, diff: 95 B (size-diff:95)
  lib/systemd/journald.conf.d/journald.conf: 143 B -> 144 B, diff: 1 B (size-diff:1)
  lib/systemd/network/90-usb-server.network: 145 B -> 133 B, diff: -12 B (size-diff:-12)
  lib/systemd/system/cc-hal-can.service: 435 B -> 436 B, diff: 1 B (size-diff:1)
  lib/systemd/system/cc-hal-modem.service: 606 B -> 607 B, diff: 1 B (size-diff:1)
  lib/systemd/system/cc-hal-pm.service: 526 B -> 527 B, diff: 1 B (size-diff:1)
  lib/systemd/system/cc-hal-update.service: 589 B -> 590 B, diff: 1 B (size-diff:1)
  lib/systemd/system/ecu-sim.service: 462 B -> 438 B, diff: -24 B (size-diff:-24)
  lib/systemd/system/sti-eol-test.service: 271 B -> 272 B, diff: 1 B (size-diff:1)
  lib/systemd/system-preset/98-cc-hal.preset: 382 B -> 324 B, diff: -58 B (size-diff:-58)
  lib/systemd/system-preset/98-systemd-config.preset: 213 B -> 200 B, diff: -13 B (size-diff:-13)
Grouped files:
Totals:
 unchanged files size   : 62.30 MB (65327929 B)
 new files size         : 4.33 MB (4540814 B) (zstd: 0 B)
   grouped : 0 B (zstd: 0 B)
   single  : 4.33 MB (4540814 B) (zstd: 0 B)
 removed files size     : 181.21 KB (185558 B) (zstd: 0 B)
   grouped : 0 B (zstd: 0 B)
   single  : 181.21 KB (185558 B) (zstd: 0 B)
 updated files size     : 7.09 MB (7430222 B) -> 6.73 MB (7060844 B) (size-diff: -360.72 KB (-369378 B))
 total diff size        : 11.06 MB (11601658 B) ()
```

File: bin/rootfs-diff.ts [hunk 1]
```diff
@@ -700,16 +734,19 @@ async function main(argv: string[]): Promise<number> {
...
-    console.log(
-      `  ${updatedFile.to.path}${fromPath}: ${updatedFile.from.size} -> ${updatedFile.to.size} (${diffStats})`
-    )
-    print(updatedFile.diffoscope.replace(/(^|\n)(.)/gm, '$1  $2'), !useDiffoscope)
+    const hFromSize = formatBytes(updatedFile.from.size)
+    const hToSize = formatBytes(updatedFile.to.size)
+    const hSizeDiff = formatBytes(updatedFile.sizeDiff)
+    // example: etc/build: 12.36 MB (12360000 B) -> 12.34 MB (12340000 B), diff: 0.02 MB (20000 B)
+    console.log(`  ${updatedFile.to.path}${fromPath}: ${hFromSize} -> ${hToSize}, diff: ${hSizeDiff} (${diffStats})`)
+    // print(updatedFile.diffoscope.replace(/(^|\n)(.)/gm, '$1  $2'), !useDiffoscope)
```
Comment (tlbdk) [inline bin/rootfs-diff.ts:746]: Just add:
```
if(useDiffoscope) {
  console.log(updatedFile.diffoscope.replace(/(^|\n)(.)/gm, '$1  $2'))
}
```

File: bin/rootfs-diff.ts [hunk 2]
```diff
@@ -700,16 +734,19 @@ async function main(argv: string[]): Promise<number> {
...
   }
 
-  print(`Grouped files:`, flags.hideGroups)
+  console.log(`Grouped files:`)
```
Comment (tlbdk) [inline bin/rootfs-diff.ts:749]: Same

File: bin/rootfs-diff.ts [hunk 3]
```diff
@@ -739,10 +776,13 @@ async function main(argv: string[]): Promise<number> {
           continue
         }
 
-        totalDiffStats += `, ${compressorName}: ${totalCompressedSize}`
+        const hTotalCompressedSize = formatBytes(totalCompressedSize)
+        totalDiffStats += `, ${compressorName}: ${hTotalCompressedSize}`
       }
 
-      print(`  ${groupRegex}: (size: ${totalSize}${totalDiffStats})`, flags.hideGroups)
+      const hTotalSize = formatBytes(totalSize)
```
Comment (tlbdk) [inline bin/rootfs-diff.ts:783]: Same

File: bin/rootfs-diff.ts [hunk 4]
```diff
@@ -753,9 +793,10 @@ async function main(argv: string[]): Promise<number> {
             continue
           }
 
-          diffStats += `, ${compressorName}: ${compressedSize}`
+          const hCompressedSize = formatBytes(compressedSize)
+          diffStats += `, ${compressorName}: ${hCompressedSize}`
         }
-        print(`    ${file.to.path}: (size: ${file.to.size}${diffStats})`, flags.hideGroups)
```
Comment (tlbdk) [inline bin/rootfs-diff.ts:758]: Same

**Review by tlbdk — CHANGES_REQUESTED** [review]

**Review by tlbdk — APPROVED** [review]

**Pull request merged**